tools/serverpod_cli/lib/src/analyzer/models/validation/restrictions.dart

```diff
@@ -228,7 +228,8 @@ class Restrictions {
       ];
     }
 
-    var parentClass = parsedModels.findByClassName(parentClassName);
+    var parentClass =
+        parsedModels.findByClassName(parentClassName) as ClassDefinition?;
```

Comment:
Question: Is this cast to `ClassDefinition?` required? Seems like it was not needed before so we should be able to skip it here?